cypress/support/auth.ts

```diff
@@ -117,7 +117,7 @@ Cypress.Commands.add('awxLogout', () => {
   cy.getByDataCy('account-menu')
     .click()
     .then(() => {
-      cy.intercept('GET', awxAPI`logout/`).as('logout');
+      cy.intercept('GET', `/api/logout/`).as('logout');
```

Comment:
Will this work against a downstream build?

Reply:
I have asked about it James Talton and he said this login and logout is not used in downstream. So I hope :-)